conf/broker.conf

```diff
@@ -514,9 +514,18 @@ delayedDeliveryEnabled=true
 
 # Control the tick time for when retrying on delayed delivery,
 # affecting the accuracy of the delivery time compared to the scheduled time.
+# Note that this time is used to configure the HashedWheelTimer's tick time for the
+# InMemoryDelayedDeliveryTrackerFactory (the default DelayedDeliverTrackerFactory).
 # Default is 1 second.
 delayedDeliveryTickTimeMillis=1000
 
+# When using the InMemoryDelayedDeliveryTrackerFactory (the default DelayedDeliverTrackerFactory), whether
+# the deliverAt time is strictly followed. When false, messages may be sent to consumers before the
+# deliverAt time by as much as the tickTimeMillis. When true, messages will not be sent to consumer until
+# the deliverAt time has passed, and they may be as late as the deliverAt time plus the tickTimeMillis for the
+# topic plus the delayedDeliveryTickTimeMillis.
+isDelayedDeliveryDeliverAtTimeStrict=false
```

Comment:
It's better to add some information about set to false which allows sending to consumers in advance for tick time but can reduce the overhead of just maintaining the delayed index in a very short time period. Otherwise, users can't find any reason to set this to false and defaults to false too.

Reply:
Done.